app/carbonapi/http_handlers.go

```diff
@@ -173,7 +173,7 @@ func (app *App) renderHandler(w http.ResponseWriter, r *http.Request) {
 		logAsError = true
 		return
 	}
-	if form.from32 == form.until32 {
+	if form.from32 >= form.until32 {
```

Comment:
Error message in the next line need to be adjusted as well in order to represent this code change

Reply:
For simplicity you can use "Invalid or empty time range"  but if this message is a part of HTTP error response than it make sense to be more verbose. 
Excessive verbosity in the logs isn't needed

Reply:
Added more precise error text there